executor/index_lookup_join_test.go

```diff
@@ -142,3 +143,77 @@ func (s *testSuite5) TestIndexJoinPartitionTable(c *C) {
 	tk.MustExec("insert into t values(1, 27, 2)")
 	tk.MustQuery("SELECT /*+ TIDB_INLJ(t1) */ count(1) FROM t t1 INNER JOIN (SELECT a, max(c) AS c FROM t WHERE b = 27 AND a = 1 GROUP BY a) t2 ON t1.a = t2.a AND t1.c = t2.c WHERE t1.b = 27").Check(testkit.Rows("1"))
 }
+
+type testSuite9 struct {
+	*baseTestSuite
+
+	is       infoschema.InfoSchema
+	testData testutil.TestData
+}
+
+func (s *testSuite9) SetUpSuite(c *C) {
+	var err error
```

Comment:
the `baseTestSuite` has one `SetUpSuite` and the `store` and `domain` is initialized in it.